modules/installation-requirements-user-infra-ibm-z-kvm.adoc

```diff
@@ -0,0 +1,185 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_ibm_z/installing-ibm-z-kvm.adoc
+
+
+[id="installation-requirements-user-infra_{context}"]
+= Machine requirements for a cluster with user-provisioned infrastructure
+
+For a cluster that contains user-provisioned infrastructure, you must deploy all
+of the required machines.
+
+One or more KVM host machines based on {op-system-base} 8.3 or later. Each {op-system-base} KVM host machine must have libvirt installed and running. The virtual machines are provisioned under each {op-system-base} KVM host machine.
+
+
+[id="machine-requirements_{context}"]
+== Required machines
+
+The smallest {product-title} clusters require the following nodes:
+
+* One temporary bootstrap machine
+
+* Three control plane, or master, machines
+
+* At least two compute machines, which are also known as worker machines
+
+[NOTE]
+====
+The cluster requires the bootstrap machine to deploy the {product-title} cluster
+on the three control plane machines. You can remove the bootstrap machine after
+you install the cluster.
+====
+
+[IMPORTANT]
+====
+To improve high availability of your cluster, distribute the control plane machines over different {op-system-base} instances on at least two physical machines.
+====
+
+The bootstrap, control plane, and compute machines must use the {op-system-first} as the
+operating system.
+
+See link:https://access.redhat.com/articles/rhel-limits[Red Hat Enterprise Linux technology capabilities and limits].
+
+[id="network-connectivity_{context}"]
+== Network connectivity requirements
+
+The {product-title} installer creates the Ignition files, which are necessary for all the {op-system-first} virtual machines. The automated installation of {product-title} is performed by the bootstrap machine.  It starts the installation of {product-title} on each node, starts the Kubernetes cluster, and then finishes. During this bootstrap, the virtual machine must have an established network connection either through a DHCP Server or static IP address.
```

Comment:
s/bootstrap machine.  It starts /bootstrap machine. It starts 

Similar suggestion: 
s/DHCP/Dynamic Host Configuration Protocol (DHCP) 

And maybe IP too, though it's common enough it may be fine.